sale_order_type/__manifest__.py

```diff
@@ -1,51 +1,38 @@
-# -*- encoding: utf-8 -*-
-##############################################################################
-#                                                                            #
-#  OpenERP, Open Source Management Solution.                                 #
-#                                                                            #
-#  @author Carlos Sánchez Cifuentes <csanchez@grupovermon.com>               #
-#                                                                            #
-#  This program is free software: you can redistribute it and/or modify      #
-#  it under the terms of the GNU Affero General Public License as            #
-#  published by the Free Software Foundation, either version 3 of the        #
-#  License, or (at your option) any later version.                           #
-#                                                                            #
-#  This program is distributed in the hope that it will be useful,           #
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of            #
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the              #
-#  GNU Affero General Public License for more details.                       #
-#                                                                            #
-#  You should have received a copy of the GNU Affero General Public License  #
-#  along with this program. If not, see <http://www.gnu.org/licenses/>.      #
-#                                                                            #
-##############################################################################
+# -*- coding: utf-8 -*-
+# Copyright 2015 Carlos Sánchez Cifuentes <csanchez@grupovermon.com>
+# Copyright 2015-2016 Oihane Crucelaegui <oihane@avanzosc.com>
+# Copyright 2015-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 Lorenzo Battistini
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    "name": "Sale Order Types",
-    "version": "8.0.1.0.1",
+    "name": "Sale Order Type",
+    "version": "10.0.1.0.0",
     "category": "Sales Management",
-    "author": "OdooMRP team, "
-              "Grupo Vermon, "
-              "AvanzOSC, "
-              "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
+    "author": "Grupo Vermon,"
+              "AvanzOSC,"
+              "Tecnativa,"
+              "Agile Business Group,"
+              "Niboo,"
               "Odoo Community Association (OCA)",
     "website": "http://www.odoomrp.com",
```

Comment:
I'd use https://github.com/OCA/sale-workflow and also can you add a short summary?